my_int_param: 3.0
my_vec_param: [0.342276, 0.201453, 0.45627]